wpilibc/src/main/native/cpp/MotorSafetyHelper.cpp

```diff
@@ -17,22 +18,22 @@
 
 using namespace frc;
 
-std::set<MotorSafetyHelper*> MotorSafetyHelper::m_helperList;
-wpi::mutex MotorSafetyHelper::m_listMutex;
+static wpi::SmallPtrSet<MotorSafetyHelper*, 32> helperList;
```

Comment:
This PR is going to be fun to redo in #562. lol

Reply:
I'll merge as a rebase so it would be easy to revert just this one commit if you wanted

Reply:
I don't think there's really a way to make it cleaner via history. Since MotorSafetyHelper is deleted entirely in #562 with stuff from that incorporated into MotorSafety, there's no way for Git to know what to do about it.